what4/src/What4/Config.hs

```diff
@@ -851,12 +854,15 @@ ppConfigLeaf nm (ConfigLeaf sty ref help) =
 configHelp ::
   Text ->
   Config ->
-  IO [Doc]
+  IO [Doc ()]
 configHelp prefix (Config cfg) =
   do m <- readIORef cfg
      let ps = Text.splitOn "." prefix
-         f :: [Text] -> ConfigLeaf -> WriterT (Seq Doc) IO ConfigLeaf
+         f :: [Text] -> ConfigLeaf -> WriterT (Seq (Doc ())) IO ConfigLeaf
          f nm leaf = do d <- liftIO (ppConfigLeaf nm leaf)
                         tell (Seq.singleton d)
                         return leaf
      toList <$> (execWriterT (traverseSubtree ps f m))
+
+text :: String -> Doc ann
```

Comment:
It doesn't look like this is exported from this file, and I only notice it being used once or twice in your changes, with all the other locations just using `pretty` directly; should we convert the rest of the uses of `text` to `pretty` and remove this?

Reply:
Yeah, I already removed similar functions from the saw-core PR, and I was planning to do that here too.